acceptance-tests/src/test/resources/features/UKDrivingLicence/DrivingLicenceCRIAPI.feature

```diff
@@ -30,7 +30,7 @@ Feature: DrivingLicence CRI API
     And Driving Licence user gets a session-id
     When Driving Licence user sends a POST request to Driving Licence endpoint using jsonRequest DVLAInvalidJsonPayload
     Then Driving Licence check response should contain Retry value as true
-    When Driving Licence user sends a POST request to Driving Licence endpoint using jsonRequest DVLAValidJsonPayload
+    When Driving Licence user sends a POST request to Driving Licence endpoint using jsonRequest DVLAValidJsonPayload and document checking route is dcs
```

Comment:
We shouldnt need this document route should be DCS by default

Reply:
Corrected.